rebench.conf

```diff
@@ -261,6 +264,8 @@ experiments:
                 benchmark: ruby-indy
             - RBX314:
                 benchmark: ruby-rbx
+            - Topaz:
+                benchmark: ruby-rbx
```

Comment:
I added the clock_gettime api and monotonic clock, now the default harness worky